Library/Homebrew/shims/linux/super/cc

```diff
@@ -189,7 +189,8 @@ class Cmd
     when /^-R(.+)?/
       # -R is not the correct way to pass rpaths to the linker
       val  = chuzzle($1) || enum.next
-      wl = "-Wl," unless mode == :ld
+      path = canonical_path(val)
+      wl   = "-Wl," unless mode == :ld
```

Comment:
Yipes. It looks as though this chunk uses `path`, which is never assigned. Does this work?
```diff
diff --git a/Library/Homebrew/shims/linux/super/cc b/Library/Homebrew/shims/linux/super/cc
index 13b6b502b..647914c7c 100755
--- a/Library/Homebrew/shims/linux/super/cc
+++ b/Library/Homebrew/shims/linux/super/cc
@@ -188,7 +188,7 @@ class Cmd
       args << "-L#{val}" if keep?(path) && @lset.add?(path)
     when /^-R(.+)?/
       # -R is not the correct way to pass rpaths to the linker
-      val  = chuzzle($1) || enum.next
+      path  = chuzzle($1) || enum.next
       wl = "-Wl," unless mode == :ld
       args << "#{wl}-rpath=" if keep?(path)
     else
```